### DISCUSSION TITLE
Lavalink v3 deprecation notice

### DISCUSSION THREAD

**Discussion by topi314** [lavalink-devs/Lavalink#1100] [Announcements]
Hi Lavalink people,

> [!CAUTION]
> As of **November 1, 2024** we will officially drop support for Lavalink v3.

A final Lavalink v3 release (`3.8.0`) will follow within the next days which will add support for the [new voice encryption modes](https://discord.com/developers/docs/change-log#voice-encryption-modes). These will be required as of **November 18, 2024** by Discord and older versions of Lavalink will stop working then.

## What does this mean?
* Lavalink v3 won't receive any fixes & new features
* We won't be offering support with issues regarding Lavalink v3 in our issue tracker or Discord

## How to prepare
* Make sure your Lavalink Client supports Lavalink v4, if not see [here](https://lavalink.dev/clients.html) for a list of compatible clients
* Download the latest Lavalink v4 release from [GitHub Releases](https://github.com/lavalink-devs/Lavalink/releases/latest)
* Update your `application.yml` to the latest example. See [here](https://lavalink.dev/configuration/index.html)

> [!TIP]
> If you run into any issues updating feel free to open a [GitHub discussion](https://github.com/lavalink-devs/Lavalink/discussions/new?category=q-a) or join the Lavalink [Discord Server](https://discord.gg/ZW4s47Ppw4) for help.

If you have any questions or concerns regarding this topic feel free to ask under this post.

**Comment by Natemo2625** [comment]
Just curious, does Lavalink v3 still work? In the process of upgrading to v4 but am hoping I can have it working temporarily until I'm done

**Reply by topi314** [reply]
Lavalink v3 itself does still work yes.
Some sources may not work anymore tho (YouTube which requires an external plugin)

**Reply by topi314** [reply]
As of march 2nd Lavalink v3 does not work anymore since DAVE is a requirement now.
For more information see: https://docs.discord.com/developers/change-log#deprecating-non-e2ee-voice-calls & https://github.com/lavalink-devs/Lavalink/discussions/1192
